services/telegram/service.go

```diff
@@ -108,7 +119,8 @@ func (s *Service) Alert(chatId, parseMode, message string, disableWebPagePreview
 		return err
 	}
 
-	resp, err := http.Post(url, "application/json", post)
+	client := s.clientValue.Load().(*http.Client)
```

Comment:
I'd probably just use a mutex, but if you keep with the `atomic.Value` and you do something like

```go
client, ok := s.clientValue.Load().(*http.Client)

if !ok {
    return <some error here>
}
```